portfolio/pom.xml

```diff
@@ -72,6 +75,52 @@
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
+      <!-- <plugin>
```

Comment:
Let's remove the commented out code from this PR before you merge it into master.